modules/reindex/src/test/java/org/elasticsearch/index/reindex/AsyncBulkByScrollActionTests.java

```diff
@@ -743,9 +742,8 @@ protected DummyAbstractBulkByScrollRequest self() {
 
         @Override
         @SuppressWarnings("unchecked")
-        protected <Request extends ActionRequest, Response extends ActionResponse,
-                RequestBuilder extends ActionRequestBuilder<Request, Response>> void doExecute(
-            Action<Response> action, Request request, ActionListener<Response> listener) {
+        protected <Request extends ActionRequest, Response extends ActionResponse>
+        void doExecute(Action<Response> action, Request request, ActionListener<Response> listener) {
```

Comment:
This formatting looks a bit weird to me, maybe switch to the same format as the next file instead?

Reply:
I actually prefer this formatting to the one in FilterClient, as it maintains the return type in an easily viewable place. The FilterClient example also at first glance looks like the parameters are local variables because of indentation, until you notice the curly at the end of the line. Given that reasoning, would you be happy if I still made all these cases the same formatting?

Reply:
Yeah. I see your point. Let's just make it more consistent then.